helm/hpcc/values.schema.json

```diff
@@ -422,6 +427,20 @@
         }
       }
     },
+    "cassandra": {
+      "type": "object",
+      "required": [ "name" ],
+      "additionalProperties": { "type": ["integer", "string", "boolean"] },
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the cassandra process"
+        },
+        "image": {
```

Comment:
is this used?

Reply:
I suspect I put it in with the idea that you could pick the cassandra version... It's been a while though